### ISSUE TITLE
Need methodology to relate segmented field polygons to ground truth polygons

### ISSUE THREAD

**Issue by geojamesc** [cropmapteam/Scotland-crop-map#40]
Since we are anticipating that the polygons geometries that the segmentation process will produce will be different from the polygons geometries that are recorded in the ground truth dataset. So we need to come up with some kind of spatial methodology that can be used to relate the two.

**Comment by Kostassan** [comment]
If there is a shapefile for the ground truth dataset we can relate them based on geographical position. 
If I can get a hold on he shapefile I can come up with something, most probably by dissolving one with the other.

**Comment by geojamesc** [comment]
on the vm there is:

/data/Ground_Truth_Polys/Export_2018_land_cover.shp

which is a shapefile of the 70,000 or so ground truth polygons that richard was handed on a usb stick in a field whilst on the farm visit!

**Comment by Kostassan** [comment]
Hi, James just looked at the land cove .shp.
My guess is the Land cover data are based in Ordnance Survey Mastermap polygons.
The segmentation results with the current settings there will not be as detailed as that.
A simple and quick solution will be to incorporate the land cover data in the segmentation shapefile, replacing the polygons from the segmentation that fall inside the land cover data.

**Comment by geojamesc** [comment]
I`m not sure about the origins of the polygons - I think they might be captured in the field. Richard do you know?. In any case what you propose to relate the two sets of polygons sounds sensible.

**Comment by Kostassan** [comment]
I have tried it yesterday to amock segmentation shapefile with some good results. My question for the origin of the polygons, if they are based in Mastermap polygons (their geometry) we should probably add a disclaimer or mention in the final documentation or map of the product.

**Comment by geojamesc** [comment]
I mentioned this at the crop map standup yesterday. According to Stephen, the polygons are captured in the field rather than being based on MasterMap.

**Comment by spatial-scales** [comment]
Yes, the polygons are captured in the field but they're often snapped to MasterMap boundaries. In these cases OS are pretty touchy about re-publishing their data as they say it is too similar to one of their products. They are the same with postcode boundaries which are digitised by NRS but snapped to MasterMap in many places, and only publish these under OSMA.

Therefore, since we'll be publishing the data including the field geometries as open data, we probably can't incorporate the land parcels into the segmentation algorithm.

**Comment by Kostassan** [comment]
So I guess we won't relate them to the segmentation shapefile. As the only way to relate the data from the ground truth polygons is through their geometry.

**Comment by spatial-scales** [comment]
Sorry Kostas, misunderstanding/misreading on my part, I thought you were suggesting that we incorporate the geometries of the land cover polygons into the segmentation process. If I now understand correctly, you're suggesting we do some sort of spatial join between the land cover polygons and the segmentation polygons to attach the land cover attributes to the segmented polygons. If so I agree that this is the best way to go, and I'll be figuring out how to do that tomorrow.
